src/main/java/health/ere/ps/service/connector/certificate/CardCertReadExecutionService.java

```diff
@@ -45,13 +45,15 @@ void init() throws Exception {
         BindingProvider bp = (BindingProvider) certificateService;
         
         bp.getRequestContext().put(BindingProvider.ENDPOINT_ADDRESS_PROPERTY,
-        endpointDiscoveryService.getCertificateServiceEndpointAddress());
-        
-        if (endpointDiscoveryService.getConnectorTlsCertTrustStore().isPresent()) {
-            String path = endpointDiscoveryService.getConnectorTlsCertTrustStore().get();
-            secretsManagerService.configureSSLTransportContext(path, endpointDiscoveryService.getConnectorTlsCertTrustStorePwd(),
-                    SecretsManagerService.SslContextType.TLS, SecretsManagerService.KeyStoreType.PKCS12, bp);
-        }
+                endpointDiscoveryService.getCertificateServiceEndpointAddress());
+
+        secretsManagerService.configureSSLTransportContext(endpointDiscoveryService.getConnectorTlsCertAuthStoreFile().orElse(null),
```

Comment:
This call should be in separated method  in the SecretsManagerService, because the first 6 parameters are always the same in the calling services.